lib/ansible/executor/module_common.py

```diff
@@ -731,7 +740,9 @@ def modify_module(module_name, module_path, module_args, task_vars=dict(), modul
 
     (module_data, module_style, shebang) = _find_snippet_imports(module_name, module_data, module_path, module_args, task_vars, module_compression)
 
-    if shebang is None:
+    if module_style == 'binary':
+        return (module_path, module_data, module_style, shebang)
```

Comment:
Probably not.  I just need to ensure we have the path available to us at all places where it is needed.